src/histolab/slide.py

```diff
@@ -266,37 +269,16 @@ def resampled_array(self, scale_factor: int = 32) -> np.array:
         ----------
         scale_factor : int, optional
             Image scaling factor. Default is 32.
+
         Returns
         ----------
         resampled_array: np.ndarray
             Resampled array
         """
         return self._resample(scale_factor)[1]
 
-    def save_scaled_image(self, scale_factor: int = 32) -> None:
-        """Save a scaled image in the correct path
-
-        Parameters
-        ----------
-        scale_factor : int, optional
-            Image scaling factor. Default is 32.
-        """
-        os.makedirs(self._processed_path, exist_ok=True)
-        img = self._resample(scale_factor)[0]
-        img.save(self.scaled_image_path(scale_factor))
-
-    def save_thumbnail(self) -> None:
-        """Save a thumbnail in the correct path"""
-        os.makedirs(self._processed_path, exist_ok=True)
-
-        img = self._wsi.get_thumbnail(self._thumbnail_size)
-
-        folder = os.path.dirname(self.thumbnail_path)
-        pathlib.Path(folder).mkdir(exist_ok=True)
-        img.save(self.thumbnail_path)
-
-    def scaled_image_path(self, scale_factor: int = 32) -> str:
-        """Return slide image path.
```

Comment:
do you want to remove all deprecated things here or in a separate PR?

Reply:
i think here is safe

Reply:
I couldn't remove `_biggest_regions` of `Slide` because it is needed by `locate_biggest_tissue_box()`. The latter method is going to be removed when we have a method in `BinaryMask` that lets you visualize the outline of the extraction mask on the slide.

Reply:
mmm what do you suggest? Wanna do this in another PR or wanna you do this change in here?

Reply:
I would do this in a separate PR

Reply:
alright ... let's do this in a furhter one